core/include/list.h

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (C) 2016 Kaspar Schleiser <kaspar@schleiser.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+ /**
+ * @addtogroup  core_util
+ * @{
+ *
+ * @file
+ * @brief       Intrusive linked list
```

Comment:
Sure you mean intrusive (german: 'aufdringlich' or 'zudringlich')? You might mean 'intrinsic' or similar?